app/webpacker/css/admin_v3/all.scss

```diff
@@ -111,7 +111,7 @@
 @import "../admin/question-mark-tooltip";
 @import "../admin/relationships";
 @import "../admin/reports";
-@import "../admin/select2";
+@import "components/select2"; // admin_v
```

Comment:
An aside, I think we might have some overlap between `admin/plugins/select2` and `admin/select2`. I can't see any reason for these to be separate files.
Perhaps we can consider cleaning that up.

Reply:
Right. I've copy/pasted file `admin/plugins/select2` content into `components/select2` file for v3 version.